config/jwt.php

```diff
@@ -120,7 +120,7 @@
     |
     */
 
-    'refresh_ttl' =>  20160,
+    'refresh_ttl' =>  2,
```

Comment:
if this is the time of the refresh token then you should have to increase the value of this as while the access token expires same time the refresh token will also expire in the same time and there is not benefit of doing this.